spec/support/request/split_checkout_request_helper.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module SplitCheckoutRequestsHelper
+  def have_split_checkout_details
+    have_content "Your details"
+  end
+
+  def split_checkout_as_guest
```

Comment:
If we've created a new/separate class (`SplitCheckoutRequestsHelper`) and included it in the spec, we don't really need to preface all these methods with `split_`. `checkout_as_guest` is a lot nicer to read than `split_checkout_as_guest`.